storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -46,6 +46,7 @@ static bool archive_first_block_zero = false;
 std::atomic<bool> Redo_Log_Reader::m_error;
 lsn_t Redo_Log_Reader::checkpoint_lsn_start;
 os_offset_t Redo_Log_Reader::checkpoint_offset_start;
+IF_DEBUG(bool force_reopen = false;);
```

Comment:
better to move this near to the first usage (unless there is goto)

Reply:
This needs to be a global variable (or a member variable of this class). We change it to true at `start` and read it at `read_log_seg_8030`. So I think the declaration should stay here

Reply:
ok, didn't realize it is global. It can stay there